WcaOnRails/app/views/server_status/index.html.erb

```diff
@@ -58,6 +58,22 @@
     </div>
   </div>
 
+  <% kind_class = Rails.env.test? || Rails.env.development? || (@expires_in || 0) > ServerStatusController::CERTIFICATE_RENEW_DELAY ? "success" : "danger" %>
```

Comment:
This logic feels like it might be repeated a bit with what's in the controller already. Would it work to have the controller create a `@certificate_error` attribute, and then change this line of code to:

```ruby
<% kind_class = @certificate_error ? "danger" : "success" %>
```

I like how this is more symmetric with the way `@regulations_load_error` is handled above.